### DISCUSSION TITLE
How do I listen to callee when I make an external call?

### DISCUSSION THREAD

**Discussion by manRisen** [fonoster/yaps#477] [Q&A] [answered]
I want to make calls to a regular phone numbers (not within a private network) using fonoster, like this :

https://learn.fonoster.com/docs/getting_started/make_an_outbound_call/

Then I want to play a recorded speech to a callee,  but I don't know how can I listen to their response to handle it with speech recognition. Please provide me with related info because I didn't find any in docs. 

P.S. if you know any open source implementations of my idea it would be even greater if you will share it with me.

**Comment by BrayanMnz** [comment, marked as answer]
Hi @manRisen  - with Fonoster you can use any speech recognition with Fonoster also, there are some tested integrations with providers
like Google, IBM Watson, etc...

This documentation may help you with that https://learn.fonoster.com/docs/tutorials/using_google_speech/
If you have any further question don't hesitate to reach us. 

cc : @psanders for any additional input on this one.
